topics/imaging/tutorials/parameter-tuning/tutorial.md

```diff
@@ -116,12 +122,11 @@ You now have everything you need to build the parameter tuning workflow!
 >    - **Ground truth is uniquely labeled**: ***No***
 >    - Performance measure(s):
 >      - ***Region-based / DICE***
```

Comment:
Dice is a proper name, not an abbreviation :) https://en.wikipedia.org/wiki/Dice-Sørensen_coefficient
```suggestion
>      - ***Region-based / Dice***
```